### PR TITLE
Set CPU scaling governor for rpi

### PR DESCRIPTION
**Pull request by phil-d-wilson**
Change-type: patch
Signed-off-by: Phil Wilson <phil@balena.io>

### CODE REVIEW

File: rpi/setup.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/bash
+
+# set the CPU Scaling governor to 'performance'
+echo "Setting CPU Scaling Governor to 'performance'"
+echo 'performance' > /sys/devices/system/cpu/cpu0/cpufreq/scaling_governor
```
Comment (rahul-thakoor) [inline rpi/setup.sh:5]: We are not testing which kernel version the OS has when running this command. Does this have any effect when running on older kernel versions?

Comment (phil-d-wilson) [reply rpi/setup.sh:5]: Very good point @rahul-thakoor - but my hunch is there is no reason not to use 'performance' when we suspect the block will need it. WDYT?

Comment (rahul-thakoor) [reply rpi/setup.sh:5]: I think that makes sense. I'm only wondering if the behaviour is the same on older versions? The new kernel has a different setting as default right but the 'performance' setting will still be understood on older versions?

File: rpi/Dockerfile.template [hunk 2]
```diff
@@ -11,16 +11,21 @@ RUN install_packages wget \
     chromium-browser \
     rpi-chromium-mods \ 
     libgles2-mesa \
+    libsemanage1 \
```
Comment (rahul-thakoor) [inline rpi/Dockerfile.template:14]: @phil-d-wilson curious why we need this

**Comment by phil-d-wilson** [comment]
Superseded by version 2

**Pull request closed without merging**